links.html

```diff
@@ -31,8 +31,8 @@
         <main>
             <h1>Useful Links</h1>
             <ul>
-                <li><a href= w3.org> Html Validator</a></li>
-                <li><a href= https://realfavicongenerator.net> Favicon Generator </a></li>
+                <li><a href="https://validator.w3.org/" target="_blank"> Html Validator</a></li>
+                <li><a href="https://realfavicongenerator.net" target="_blank"> Favicon Generator </a></li>
```

Comment:
Best practice is to use double quotes around all attribute values
`target="_blank"` causes hyperlinks to open in new tab